### ISSUE TITLE
wkhtmltopdf - Error: Failed to load http

### ISSUE THREAD

**Issue by myslts** [wkhtmltopdf/wkhtmltopdf#4798]
**wkhtmltopdf version(s) affected**: 0.12.5

**OS information**
Centos 8.2

**Description**
I used Scriptcase RAD Tools, who used wkhtmltopdf to make pdf file from html format. I have the html file and inside html I used css also. When generating pdf, it shows error.

**How to reproduce**
wkhtmltopdf --page-size A4 --orientation Portrait --header-right "[page]" http://url:81/nms/_lib/tmp/sc_grid_lokasi_layanan_html_nf956kn6oudbmqstc48trkljhk.html /var/www/html/nms/_lib/tmp/sc_pdf_20200810090001_928_grid_lokasi_layanan.pdf

and it will return:
Error: Failed to load http://url:81/nms/_lib/tmp/sc_grid_lokasi_layanan_html_nf956kn6oudbmqstc48trkljhk.html, with network status code 1 and http status code 0 - Connection refused
Error: Failed loading page http://url:81/nms/_lib/tmp/sc_grid_lokasi_layanan_html_nf956kn6oudbmqstc48trkljhk.html (sometimes it will work just to ignore this error with --load-error-handling ignore)
Exit with code 1 due to network error: ConnectionRefusedError

If I execute the source using system path, like $ wkhtmltopdf --page-size A4 --orientation Portrait --header-right "[page]" /var/www/html/nms/_lib/tmp/sc_grid_lokasi_layanan_html_nf956kn6oudbmqstc48trkljhk.html /var/www/html/nms/_lib/tmp/sc_pdf_20200810090001_928_grid_lokasi_layanan.pdf
it works well...

**Expected behavior**
I try to open the link http://url:81/nms/_lib/tmp/sc_grid_lokasi_layanan_html_nf956kn6oudbmqstc48trkljhk.html using my browser, and it could shows the html page.
I guess, it's blocked by firewall, I disabled the firewall, but it is still showing error.

**Possible Solution**  
This is the second time, I deploy on Centos 8. On my previous experience, it shows error but I installed wkhtmltopdf 0.12.5 and it solved already, but this time, it didn't work.

**Comment by Zswdhy** [comment]
E:\David\company\projects\html\cart\first_test.2>wkhtmltopdf  index.html 6.pdf
Loading pages (1/6)
Warning: Blocked access to file E:/David/company/projects/html/cart/first_test.2/static/js/jquery.js
Warning: Blocked access to file E:/David/company/projects/html/cart/first_test.2/static/js/vue.js
Warning: Blocked access to file E:/David/company/projects/html/cart/first_test.2/static/js/first_part.js
Warning: Blocked access to file E:/David/company/projects/html/cart/first_test.2/static/css/index.css
Error: Failed to load about:blank, with network status code 301 and http status code 0 - Protocol "about" is unknown
Error: Failed to load about:blank, with network status code 301 and http status code 0 - Protocol "about" is unknown
Error: Failed to load about:blank, with network status code 301 and http status code 0 - Protocol "about" is unknown
Warning: Blocked access to file E:/David/company/projects/html/cart/first_test.2/images/logo.png
Error: Failed to load about:blank, with network status code 301 and http status code 0 - Protocol "about" is unknown
Error: Failed to load about:blank, with network status code 301 and http status code 0 - Protocol "about" is unknown
Counting pages (2/6)
Resolving links (4/6)
Loading headers and footers (5/6)
Printing pages (6/6)
Done
Exit with code 1 due to network error: ProtocolUnknownError

**Comment by MaisumAbbas** [comment]
> E:\David\company\projects\html\cart\first_test.2>wkhtmltopdf index.html 6.pdf
> Loading pages (1/6)
> Warning: Blocked access to file E:/David/company/projects/html/cart/first_test.2/static/js/jquery.js
> Warning: Blocked access to file E:/David/company/projects/html/cart/first_test.2/static/js/vue.js
> Warning: Blocked access to file E:/David/company/projects/html/cart/first_test.2/static/js/first_part.js
> Warning: Blocked access to file E:/David/company/projects/html/cart/first_test.2/static/css/index.css
> Error: Failed to load about:blank, with network status code 301 and http status code 0 - Protocol "about" is unknown
> Error: Failed to load about:blank, with network status code 301 and http status code 0 - Protocol "about" is unknown
> Error: Failed to load about:blank, with network status code 301 and http status code 0 - Protocol "about" is unknown
> Warning: Blocked access to file E:/David/company/projects/html/cart/first_test.2/images/logo.png
> Error: Failed to load about:blank, with network status code 301 and http status code 0 - Protocol "about" is unknown
> Error: Failed to load about:blank, with network status code 301 and http status code 0 - Protocol "about" is unknown
> Counting pages (2/6)
> Resolving links (4/6)
> Loading headers and footers (5/6)
> Printing pages (6/6)
> Done
> Exit with code 1 due to network error: ProtocolUnknownError

Try doing it from 'from_url' instead of 'from_file'.

**Comment by PhilterPaper** [comment]
Agreed, is it _really_ trying to load `http://url:81/...` or are you just obscuring the real domain? Also, some of those errors _do_ sound like you need to enable local file access (command line flag), if you're actually trying to load a local file. A recent security-related  change to wkHTMLtoPDF no longer allows local file access by default.

**Comment by clausMeko** [comment]
@PhilterPaper fixed it for me.
